core/runtime/src/main/java/io/quarkus/runtime/logging/LoggingSetupRecorder.java

```diff
@@ -327,7 +327,7 @@ public static void initializeBuildTimeLogging(LogConfig config, LogBuildTimeConf
 
         Map<String, Handler> namedHandlers = createNamedHandlers(config, consoleConfig, Collections.emptyList(),
                 Collections.emptyList(), Collections.emptyList(), Collections.emptyList(), errorManager, logCleanupFilter,
-                Collections.emptyMap(), launchMode, dummy, true);
+                Collections.emptyMap(), launchMode, dummy, false);
```

Comment:
In the build time initialization we don't want filters, as can be seen in line `323` already.
If we don't make this `false`, JBossLogManager rightly complains about not being able to find a filter.